### DISCUSSION TITLE
online vdoc: usage examples

### DISCUSSION THREAD

**Discussion by redredwine** [vlang/v#11676] [Ideas]
Hi there, I am new,  I just came across the Vlang.io site, and I was trying to learn more about it.  I've read the documentation page. and then went to the Modules page  https://modules.vlang.io/

it would be great to have examples of how to use the modules/functions. 
I was just thinking instead of writing examples of how to use the different modules listed on that page,  why not publish your unit test cases? as I am sure you have test case for every single module, every single function there that would make calls to each and every module and function, right?  That would save you a lot of time ,rather than coming up with examples. 

Thanks.

**Comment by JalonSolov** [comment]
Simplest thing for now is to look for `*_test.v` files in the source tree.  Those are the existing tests.

There are plans to do more tutorial style documents, but there is too much going on with compiler changes right now.  Examples will likely come after the 0.2 release, when the syntax, etc. is stabilized.

**Comment by nmsobri** [comment]
yep, this is badly needed..i barely understand anything when reading the built in functions..but totally understood
